packages/bulk-edit/package.json

```diff
@@ -7,6 +7,7 @@
     "@theia/editor": "1.23.0",
     "@theia/filesystem": "1.23.0",
     "@theia/monaco": "1.23.0",
+    "@theia/monaco-editor-core": "alpha",
```

Comment:
I (or actually @vince-fugnitto, who has been extremely helpful in publishing `@theia/monaco-editor-core`), have tagged the new Monaco for this package as an `alpha` release so that people pulling `latest` doesn't get the new package and so that we can make changes to the Monaco release and then change only the `yarn.lock` entry here for further testing.